[2025-04-02T23:00:05.953+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Weather_Proj.Write_data_in_kafka manual__2025-04-02T22:59:45.123682+00:00 [queued]>
[2025-04-02T23:00:05.960+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Weather_Proj.Write_data_in_kafka manual__2025-04-02T22:59:45.123682+00:00 [queued]>
[2025-04-02T23:00:05.961+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-04-02T23:00:05.973+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): Write_data_in_kafka> on 2025-04-02 22:59:45.123682+00:00
[2025-04-02T23:00:05.977+0000] {standard_task_runner.py:60} INFO - Started process 2008 to run task
[2025-04-02T23:00:05.981+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Weather_Proj', 'Write_data_in_kafka', 'manual__2025-04-02T22:59:45.123682+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/Project_tasks.py', '--cfg-path', '/tmp/tmpa_zev4t0']
[2025-04-02T23:00:05.983+0000] {standard_task_runner.py:88} INFO - Job 23: Subtask Write_data_in_kafka
[2025-04-02T23:00:06.026+0000] {task_command.py:423} INFO - Running <TaskInstance: Weather_Proj.Write_data_in_kafka manual__2025-04-02T22:59:45.123682+00:00 [running]> on host 520607ad0646
[2025-04-02T23:00:06.100+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Weather_Proj' AIRFLOW_CTX_TASK_ID='Write_data_in_kafka' AIRFLOW_CTX_EXECUTION_DATE='2025-04-02T22:59:45.123682+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-02T22:59:45.123682+00:00'
[2025-04-02T23:00:07.109+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-02T23:00:07.120+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Weather_Proj, task_id=Write_data_in_kafka, execution_date=20250402T225945, start_date=20250402T230005, end_date=20250402T230007
[2025-04-02T23:00:07.155+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-02T23:00:07.179+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
